winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+
+### Customize the cells based on Column Name
+
+You can customize the cells based on `DataGridCellExcelExportOptions.ColumnName` property in the `CellsExportHandler`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private static void CellsExportHandler(object sender, DataGridCellExcelExportOptions e)
+{
+
+    if (e.ColumnName != "OrderID")
+        return;
+
+    e.Range.CellStyle.Font.Size = 12;
+    e.Range.CellStyle.Font.Color = ExcelKnownColors.Pink;
+    e.Range.CellStyle.Font.FontName = "Segoe UI";
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-cell-value-based-on-column.png" alt="Customizing Cell Values based on Column name while Exporting to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+Here, `OrderID` column cells are customized while exporting.
+
+## Exporting DetailsView
+
+By default, [DetailsViewDataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.DetailsViewDataGrid.html) will be exported to Excel. You can customize its exporting operation by using `DetailsViewExportHandler`.
+
+### Excluding DetailsViewDataGrid while exporting
+
+You can exclude particular `DetailsViewDataGrid` while exporting, by using the `DetailsViewExportHandler` and [DataGridDetailsViewExcelExportOptions.Cancel](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel) .
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.DetailsViewExportHandler = DetailsViewExportHandler;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private static void DetailsViewExportHandler(object sender, DataGridDetailsViewExcelExportOptions e)
+{
+    var recordEntry = e.NodeEntry as RecordEntry;
+    var record = (recordEntry.Data as Model);
+
+    if (record != null && record.OrderID == 1002)
+        e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-data-in-detailsview.png" alt="Excluding Specific DetailsView while Exporting to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+Here, `DetailsViewDataGrid` is not exported for the parent record having `OrderID` as 1002.
+
+### Excluding DetailsViewDataGrid columns from exporting
+
+You can exclude [DetailsViewDataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.DetailsViewDataGrid.html) columns while exporting, by using `DetailsViewExportHandler` and `DataGridDetailsViewExcelExportOptions.ExcludedColumns`.
```

Comment:
Exclude the [DetailsViewDataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.DetailsViewDataGrid.html) columns while exporting by using the `DetailsViewExportHandler` and `DataGridDetailsViewExcelExportOptions.ExcludedColumns.`

Reply:
Content modified